src/conductor.ts

```diff
@@ -73,80 +67,51 @@ export class Conductor {
 
   awaitClosed = () => this._wsClosePromise
 
-  cellId = (appId: string, nick: CellNick): CellId => {
-    const cellId = this._cellIds[appId][nick]
-    if (!cellId) {
-      throw new Error(`Unknown cell nickname: ${nick} in app: ${appId}`)
-    }
-    return cellId
-  }
-
-  installApp = async (agent_key: AgentPubKey, app: T.HappBundle) => {
-    const installAppReq : InstallAppRequest = {
-      app_id: app.id,
-      agent_key,
-      dnas: app.dnas
+  // this function will auto-generate an `app_id` and
+  // `dna.nick` for you, to allow simplicity
+  installHapp = async (agentPubKey: AgentPubKey, agentHapp: T.AgentHapp): Promise<T.InstalledAgentHapp> => {
+    // account for simple case where AgentHapp is just a single DNA
+    // and thus putting it into an array feels bloated
+    const dnaPaths: T.DnaPath[] = typeof agentHapp === 'string' ? [agentHapp] : agentHapp
+    const installAppReq: InstallAppRequest = {
+      app_id: `app-${uuidGen()}`,
+      agent_key: agentPubKey,
+      dnas: dnaPaths.map((dnaPath, index) => ({
+        path: dnaPath,
+        nick: `${index}${dnaPath}-${uuidGen()}`
+      }))
     }
-    const {cell_data: cellData} = await this.adminClient!.installApp(installAppReq)
-    // save the returned cellIds and cellNicks for later reference
-    for (const installedCell of cellData) {
-      const [cellId, cellNick] = installedCell
-      this._cellIds[app.id][cellNick] = cellId
-    }
-  }
-
-  _loadCellNicks = async (app_id: AppId) => {
```

Comment:
We can't get rid of this because it's needed for the use case of stopping and starting conductors.  In that case we have to reload the cell information into tryorama.

Reply:
This is wrong, it should actually work just fine as the cell info from the initial install should work.